guide/inheritance-wallet/succession.md

```diff
@@ -0,0 +1,229 @@
+
+Christina and David open the bag and take out the recovery PDF. On the first page they see the QR code with the wallet configuration. Both of them open our app on their phones and import the wallet by scanning the QR code. 
+
+{% include image-gallery.html pages = page.images_wallet-import %}
+
+After importing the wallet, Christina and David can view the wallet balance and transaction history.
+
+### Activating the wallet on the signing devices
+
+Next, they need to import the wallet to their respective signing devices. To do this, they follow the same process as Alice did during [cosigner onboarding]({{ '/guide/inheritance-wallet/onboarding-cosigners/#alice-activates-the-wallet-on-her-bitbox/' | relative_url }}). 
+
+### Timelock constraints
+
+After activating the wallet on their signing devices, Christina and David have successfully gained access to their inheritance. However, they cannot immediately spend the funds using their inheritance keys due to the timelock constraints set up by their parents.
+
+As we have covered on the [use case page]({{ '/guide/inheritance-wallet/overview/' | relative_url }}), the lock period for the inheritance key set is set to 12 months (6 months for the recovery path + 6 months for the inheritance key set). Since Alice and Bob's last transaction was 5 months ago, Christina and David will have to wait an additional 7 months before they can use their inheritance keys.
+
+{% include tip/open.html color="blue" icon="info" label="Faster access with primary keys" %}
+
+If Christina and David had access to their parents' signing devices, they could potentially use the recovery path after 6 months. However, they don't have this access, so they must wait for the full 12-month period. 
+
+{% include tip/close.html %} 
+
+### Distributing the inheritance
+
+After the 12-month waiting period, the inheritance key set becomes active. Christina and David meet to create the transactions that will distribute their inheritance. 
+
+Christina goes on to create the first transaction, which will send half of the funds to her own wallet.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-create %}
+
+The transaction has been created and is ready to be signed. Christina goes ahead and sings it with her hardware signing device.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-christina %} 
+
+After Christina is done, she shares the partially signed transaction with David by generating a QR code. David imports the transaction by scanning the code. After reviewing the details he adds his signature and the app broadcasts the fully signed transaction to the bitcoin network.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-david %} 
+
+While waiting for the first transaction to be confirmed, Christina and David repeat the process to create and sign a second transaction, sending the remaining 50% of the funds to David's personal wallet.
```

Comment:
In a more adversarial environment, David may not be OK with signing the transaction for Christina without any guarantee that she will also sign his transaction. In that case, Christina can create a batch transaction, or two separate transactions. That way she is committed to both transactions before David signs anything.